### PR TITLE
Feat: add interface and interaction implementation

### PR DESCRIPTION
**Pull request by wangbow**
Add interface path and interaction to App list 
Style optimization
Application list interactive implementation
Bread crumb interactive implementation

### CODE REVIEW

File: src/api/namespace.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { namespace } from './productionLink';
+import { getDomain } from '../utils/common';
+
+const baseURLOject = getDomain();
+export function getNamespaceList(params: any) {
+  const url = baseURLOject.MOCK ? namespace_dev : namespace;
+  return get(url, params).then((res) => res);
+}
+
+export function createNamespaceList(params: any) {
```
Comment (barnettZQG) [inline src/api/namespace.ts:12]:
```suggestion
export function createNamespace(params: any) {
```

File: src/api/addons.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { addons } from './productionLink';
+import { getDomain } from '../utils/common';
+const baseURLOject = getDomain();
+
+export function getAddonsList(params: any) {
+  const url = baseURLOject.MOCK ? addons_dev : addons;
+  return get(url, params).then((res) => res);
+}
+
+export function createAddons(params: any) {
```
[the remark this answers is not in the document]
Comment (hanxie-crypto) [reply src/api/addons.ts:12]: 每一个方法都要判断访问路径吗？ 能否把基础路径的判断独立出来

[the remark this answers is not in the document]
Comment (wangbow) [reply src/api/addons.ts:14]: done

File: src/api/namespace.ts (hunk 1 above)
Comment (wangbow) [reply src/api/namespace.ts]: done.按照 createNamespace 命名了

File: src/api/cluster.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { cluster } from './productionLink';
+import { getDomain } from '../utils/common';
+const baseURLOject = getDomain();
+
+export function getClusterList(params: any) {
+  const url = baseURLOject.MOCK ? cluster_dev : cluster;
+  return get(url, params).then((res) => res);
+}
+
+export function createClusterList(params: any) {
```
[the remark this answers is not in the document]
Comment (wangbow) [reply src/api/cluster.ts]: 更新为createCluster method

File: src/api/addons.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (wangbow) [reply src/api/addons.ts:14]: done

**Pull request merged**